grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/query/QueryAnswers.java

```diff
@@ -21,29 +21,22 @@
 import ai.grakn.concept.Concept;
 import ai.grakn.concept.Type;
 import ai.grakn.graql.VarName;
-import ai.grakn.graql.internal.reasoner.atom.Atom;
 import ai.grakn.graql.admin.Atomic;
-import ai.grakn.graql.internal.reasoner.atom.AtomicFactory;
 import ai.grakn.graql.internal.reasoner.atom.NotEquals;
-import ai.grakn.graql.internal.reasoner.atom.binary.Relation;
+import ai.grakn.graql.internal.reasoner.atom.binary.TypeAtom;
 import ai.grakn.graql.internal.reasoner.atom.predicate.IdPredicate;
 
 import com.google.common.collect.Maps;
-import java.util.ArrayList;
+import com.google.common.collect.Sets;
```

Comment:
This is Guava's `Sets` "helper" class - not a set implementation. It's used for `Sets#intersection` below.